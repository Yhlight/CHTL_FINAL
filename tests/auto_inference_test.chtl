div {
    id: mydiv;
    style {
        width: 100px;
    }
}

p {
    class: mypara;
    style {
        font-size: 16px;
        display: block;
    }
}

span {
    style {
        width: mydiv.width;
        font-size: mypara.font-size;
        display: p.display;
    }
}
