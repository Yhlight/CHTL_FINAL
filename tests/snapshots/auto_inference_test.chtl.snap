<div id="mydiv" style="width: 100px; " />
<p class="mypara" style="display: block; font-size: 16px; " />
<span style="display: block; font-size: 16px; width: 100px; " />
